lx_annotate/asgi.py

```
@@ -9,15 +9,31 @@
+
+print("DJANGO_SETTINGS_MODULE =", os.getenv("DJANGO_SETTINGS_MODULE"))
+print("sys.path[:10] =", sys.path[:10])
+
+m = os.getenv("DJANGO_SETTINGS_MODULE", "lx_annotate.settings.settings_prod")
+staticfiles_dir = os.getenv(
+    "DJANGO_STATIC_ROOT", os.path.join(os.path.dirname(__file__), "static")
+)
+try:
+    importlib.import_module(m)
+    print("Imported settings module OK:", m)
+except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)